docs/sources/installation/upgrading.md

```diff
@@ -146,3 +146,5 @@ If you have `login_remember_days` configured to 0 (zero) you should change your
 login_maximum_inactive_lifetime_days = 1
 login_maximum_lifetime_days = 1
 ```
+
+The default cookie name for storing the auth token is `grafana_sess`. you can configure this with `login_cookie_name` in `[auth]` settings.
```

Comment:
The default cookie name is `grafana_session`. `grafana_sess` is still used and the default cookie name for the session storage.